### PR TITLE
PD-196 fix specs p1

### PR DESCRIPTION
**Pull request by andreipope**

### CODE REVIEW

File: content/portworx-install-with-kubernetes/disaster-recovery/px-metro/1-install-px.md [hunk 1]
```diff
@@ -44,13 +44,23 @@ Specifying the same **ClusterID** and **Kvdb Endpoints** as your existing cluste
 If your Kubernetes clusters have exactly the same configuration, you can use the URL specified by the `portworx.com/install-source` annotation on the existing Portworx DaemonSet to fetch the spec for your new cluster:
 
 ```text
-apiVersion: extensions/v1beta1
```
Comment (andreipope) [inline content/portworx-install-with-kubernetes/disaster-recovery/px-metro/1-install-px.md]: kubectl apply fails with "The DaemonSet "portworx" is invalid: spec.template.spec.containers: Required value". Probably some parts are missing.

Comment (rodrigc) [reply content/portworx-install-with-kubernetes/disaster-recovery/px-metro/1-install-px.md]: Looking at that page, it looks like that is just a snippet which is used as an example. 
 It  is not a full yaml that can be applied to a Kubernetes cluster.  So this is probably OK.

A full daemonset yaml is something like:

https://install.portworx.com/2.1?mc=false&kbver=1.14.6&b=true&f=true&c=px-cluster-6a686c79-880d-4d7f-ae2f-e4227aa0ba3c&stork=true&lh=true&st=k8s

Comment (andreipope) [reply content/portworx-install-with-kubernetes/disaster-recovery/px-metro/1-install-px.md]: @rodrigc You're probably right. If so, should add something like "listing truncated for brevity".?

Comment (rodrigc) [reply content/portworx-install-with-kubernetes/disaster-recovery/px-metro/1-install-px.md]: No, I think it is OK, just leave it as is.

Comment (andreipope) [reply content/portworx-install-with-kubernetes/disaster-recovery/px-metro/1-install-px.md]: Gotcha. Thanks, @rodrigc !

**Pull request merged**